apps/api/tests/test_user_apply.py

```diff
@@ -36,6 +36,21 @@
     "frq_dream_job": "I am pkfire",
 }
 
+SAMPLE_APPLICATION_NO_JS = {
+    "first_name": "pk",
+    "last_name": "fire",
+    "pronouns": "other",
+    "ethnicity": "fire",
+    "is_18_older": "true",
+    "school": "UC Irvine",
+    "education_level": "Fifth+ Year Undergraduate",
+    "major": "Computer Science",
+    "is_first_hackathon": "false",
+    "portfolio": "https://github.com",
+    "frq_collaboration": "I am pkfire",
+    "frq_dream_job": "I am pkfire",
+}
```

Comment:
Suggestion: instead of writing out a full copy of the sample data, we could use `model_copy` and manually override fields, see `test_application_data_is_bson_encodable`